http-client/src/main/java/io/airlift/http/client/jetty/JettyIoPoolManager.java

```diff
@@ -0,0 +1,73 @@
+package io.airlift.http.client.jetty;
+
+import com.google.inject.Injector;
+import com.google.inject.Key;
+
+import javax.annotation.PreDestroy;
+import javax.inject.Inject;
+
+import java.lang.annotation.Annotation;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class JettyIoPoolManager
+{
+    private final List<JettyHttpClient> clients = new ArrayList<>();
+    private final String name;
+    private final Class<? extends Annotation> annotation;
+    private final AtomicBoolean destroyed = new AtomicBoolean();
+    private JettyIoPool pool;
+    private Injector injector;
+
+    public JettyIoPoolManager(String name, Class<? extends Annotation> annotation)
+    {
+        this.name = name;
```

Comment:
Name is no longer used

Reply:
`name` is passed to the `JettyIoPool` below.